### PR TITLE
It's 2022 bruh

### PR DESCRIPTION
**Pull request by LyQuid12**
## Description

Only update year in entire files to 2022

## How Has This Been Tested?

Yeet
## Checklist

- [x] SUS

Jk, my Geno ;)

### CODE REVIEW

**Comment by LyQuid12** [comment]
Nyanyame nyanyajyuunyanya-do no nyarabi de nyakunyaku inyanyaku nyanyahan nyanyadai nyanynaku nyarabete nyaganyagame.

Lmao, you got rick rolled by EterNomm

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2021 TheFarGG & TheGenocides
+Copyright (c) 2022 TheFarGG & TheGenocides
```
Comment (TheGenocides) [inline LICENSE:3]: Better to use 2021-present, people can see the year where the repo was started.

Comment (LyQuid12) [reply LICENSE:3]: kk

Comment (LyQuid12) [reply LICENSE:3]: just LICENSE?

Comment (TheGenocides) [reply LICENSE:3]: > just LICENSE?

All the license messages. One in [LICENSE](https://github.com/PyTweet/PyTweet/blob/master/LICENSE) file and [readme.md](https://github.com/PyTweet/PyTweet/blob/master/README.md)

Comment (LyQuid12) [reply LICENSE:3]: > > just LICENSE?
> 
> All the license messages. One in [LICENSE](https://github.com/PyTweet/PyTweet/blob/master/LICENSE) file and [readme.md](https://github.com/PyTweet/PyTweet/blob/master/README.md)

oghey

**Pull request closed without merging**